mozilla_version/gecko.py

```diff
@@ -354,6 +435,12 @@ class FirefoxVersion(_VersionWithEdgeCases):
     """Class that validates and handles Firefox version numbers."""
 
     _RELEASED_EDGE_CASES = ({
+        'major_number': 1,
+        'minor_number': 5,
+        'patch_number': 0,
+        'old_fourth_number': 1,
+        'release_candidate_number': 1,
+    }, {
```

Comment:
Nit: Don't we need a similar entry for `2.0.x.y` given https://github.com/mozilla-releng/mozilla-version/pull/63/files#diff-0a7ba49d8fa30825a6245b5e7565cedbR169?

Reply:
That's not super clear, but that 1.5.0.1rc1 entry is an edge case, even for the four-digit-rule. So far, RCs have happening on version numbers that only have 2 digits. There wasn't any case like this one in `2.0.x.y` so we don't need such an entry